scripts/linux/electron-installer-appimage.sh

```diff
@@ -102,17 +101,14 @@ else
   exit 1
 fi
 
-if [ -z "$TMPDIR" ]; then
-  TMPDIR=$(mktemp -d)
-fi
-mkdir -p "$TMPDIR"
+APPIMAGE_TMPDIR=$(mktemp --dry-run -d -t "$ARGV_APPLICATION_NAME-linux-$ARGV_ARCHITECTURE-XXXX")
```

Comment:
Haha, sounds fun! I'll update this PR with the changes in a minute (currently doing some testing to ensure I didn't break anything :) )